recipes/openmpi/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from os import path
+
+
+class OpenmpiConan(ConanFile):
+    name = "openmpi"
+    license = "3-clause BSD license"
+    author = "Conan Community"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A High Performance Message Passing Library"
+    topics = ("conan", "openmpi", "mpi")
+    homepage = "https://www.open-mpi.org/"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake"
+
+    def source(self):
+        self.output.info("Downloading OpenMPI from the mirror")
+
+        url = self.conan_data["sources"][self.version]["url"]
+        tools.get(
+            url,
+            sha1=self.conan_data["sources"][self.version]["sha1"],
+            md5=self.conan_data["sources"][self.version]["md5"],
+        )
+
+    def build(self):
+        autotools = AutoToolsBuildEnvironment(self)
+
+        folder_name = "{name}-{version}".format(
+            name=self.name, version=self.version
+        )
+
+        autotools.configure(
+            configure_dir=path.join(self.build_folder, folder_name)
+        )
+        autotools.make()
+        autotools.install()
```

Comment:
You should an install step with the [`package` method](https://docs.conan.io/en/latest/reference/conanfile/methods.html#package).

Reply:
I was not sure about this because during this step the installation is done... I'll add